### ISSUE TITLE
Problem loading modules 

### ISSUE THREAD

**Issue by Spool5520** [xanathar/moonsharp#222]
So I've read a little about modules and what not. Other issues opended on here. My problem is with loading a lua module. Not any other type of module. 
I have created a module that is a lua file that returns a table of functions.
Then I have a lua file that I use that module in. 
When I run everything in my C# application I get the error "module 'name' not found".
Both lua files are in the same directory. 

Here is the module code:
```
defaultEntites = {}

function defaultEntites.menuButton(buttontext)
	return {
				type = "button",
				text = buttontext,
				skin = "default",
				anchor = "autocenter",
				size = {x=250,y=50}
		   }
end

function  defaultEntites.menuHeader(headertext)
	return 
	{		
		type = "header",
		text = headertext,
		anchor = "topcenter",
		data = 
		{
			fillcolor = "white"
		}
	}	   
end

function  defaultEntites.menuPanel(panelID)
	return 
	{
		type = "panel",
		anchor = "center",
		skin = "default",
		size = {x=500,y=600},
		id = panelID
	}
end

return defaultEntites
```

No problems there
Here is the calling code:

`utils = require ("gui-utilities")`

So to recap. Both lua files are in the same folder. 
Just plain old lua. Nothing special.

So am I doing something wrong? Why isnt moonsharp able to locate this module?

**Comment by harmonise** [comment]
I think you need to specify the module path http://www.moonsharp.org/scriptloaders.html

`// These two lines are equivalent:
((ScriptLoaderBase)script.Options.ScriptLoader).ModulePaths = new string[] { "MyPath/?", "MyPath/?.lua" };
// or
((ScriptLoaderBase)script.Options.ScriptLoader).ModulePaths = ScriptLoaderBase.UnpackStringPaths("MyPath/?;MyPath/?.lua");`

**Comment by kylesmyrk** [comment]
It's worth mentioning that you will also actually need to define which type of script loader you're using. I spent a while wondering why I wasn't able to require modules after setting the path.

http://www.moonsharp.org/scriptloaders.html#how-to-specify-which-script-loader-to-use
